### ISSUE TITLE
🐛 [BUG] - Fix buildURL() when API url has a path

### ISSUE THREAD

**Issue by marrouchi** [Hexastack/Hexabot#114]
### Description

In case when the url has a path (example : `http://my-domain.com/api/`) the buildURL does take into consideration the path.

```ts
// frontend/src/utils/URL.ts
export const buildURL = (baseUrl: string, relativePath: string): string => {
  try {
    const url = new URL(relativePath, baseUrl);

    return url.toString();
  } catch {
    throw new Error(`Invalid base URL: ${baseUrl}`);
  }
};
```

### Reproduction steps

```bash
buildURL('/user/4732846926492', 'http://my-domain.com/api/'); 
// returns 'http://my-domain.com/user/4732846926492'
```

**Comment by mSaurabh** [comment]
@marrouchi yes please assign it to me

**Comment by marrouchi** [comment]
@mSaurabh Done

**Comment by mSaurabh** [comment]
@marrouchi, I think your example of reproduction steps needs to be corrected.

Instead of 
`buildURL('/user/4732846926492', 'http://my-domain.com/api/'); `

should it be this?
`buildURL('http://my-domain.com/api/','/user/4732846926492')`

If so, then [this PR](https://github.com/Hexastack/Hexabot/pull/117/commits/2871fb432c9f90f8c76220d6d58e6a95a2c8fc08) should fix that.

**Comment by marrouchi** [comment]
Yes it does, @devamitranjan thank you. If you could please get the issue assigned to you before working on a PR 🙏

**Comment by devamitranjan** [comment]
> Yes it does, @devamitranjan thank you. If you could please get the issue assigned to you before working on a PR 🙏

Sure @marrouchi  will do it from next ticket.
